spec.html

```diff
@@ -6596,7 +6602,8 @@ <h1>RunJobs ( )</h1>
       1. In an implementation-dependent manner, obtain the ECMAScript source texts (see clause <emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>) and any associated host-defined values for zero or more ECMAScript scripts and/or ECMAScript modules. For each such _sourceText_ and _hostDefined_, do
         1. If _sourceText_ is the source code of a script, then
           1. Perform EnqueueJob(`"ScriptJobs"`, ScriptEvaluationJob, &laquo; _sourceText_, _hostDefined_ &raquo;).
-        1. Else _sourceText_ is the source code of a module,
+        1. Else,
+          1. Assert: _sourceText_ is the source code of a module.
```

Comment:
4998520

Reply:
@ljharb: I'm curious why you think this one should be a Note rather than an Assert. The "loop header" says "source texts for scripts and/or modules", so if it's not the source of a script, it must be the source of a module.

Reply:
Mainly because it seems like prose describing the effect of being the negative case of the if - but i guess if the corresponding prose works in an if, then this prose would work as an assert.

Reply:
Manually reverted it back to an assert.